test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -129,6 +130,10 @@ export default class BasePageObject {
     return this.getForgotPasswordLink().exists;
   }
 
+  async helpLinkExists() {
```

Comment:
I think we may want change some of these methods to search for the links more semantically instead of using the `data-se` attr if possible.  @shawyu-okta what do you think?

Reply:
yeah you can use the `getLink` helper method from `BaseFormObject` which uses `getByRole`

Reply:
Done